ui/page/dcrdex/market.go

```diff
@@ -0,0 +1,937 @@
+	orderTypesDropdown  *cryptomaterial.DropDown
+	isMarketOrder       bool
+
+	priceEditor        cryptomaterial.Editor
+	switchLotsOrAmount *cryptomaterial.Switch
+	lotsAmountEditor   cryptomaterial.Editor
+	totalEditor        cryptomaterial.Editor
+
+	seeFullOrderBookBtn cryptomaterial.Button
+
+	createOrderBtn    cryptomaterial.Button
+	buyOrder          bool
+	immediateOrder    cryptomaterial.CheckBoxStyle
+	immediateMoreInfo *cryptomaterial.Clickable
+
+	openOrdersBtn       cryptomaterial.Button
+	orderHistoryBtn     cryptomaterial.Button
+	openOrdersDisplayed bool
+
+	ordersTableHorizontalScroll *widget.List
+
+	showLoader bool
+}
+
+func NewDEXMarketPage(l *load.Load) *DEXMarketPage {
+	th := l.Theme
+	pg := &DEXMarketPage{
+		Load:                        l,
+		ordersContainer:             &widget.List{List: layout.List{Axis: vertical, Alignment: layout.Middle}},
+		GenericPageModal:            app.NewGenericPageModal(DEXAccountOnboardingID),
+		scrollContainer:             &widget.List{List: layout.List{Axis: vertical, Alignment: layout.Middle}},
+		priceEditor:                 newTextEditor(l.Theme, values.String(values.StrPrice), "0", false),
+		totalEditor:                 newTextEditor(th, values.String(values.StrTotal), "", false),
+		switchLotsOrAmount:          l.Theme.Switch(), // TODO: Set last user choice, default is unchecked.
+		lotsAmountEditor:            newTextEditor(l.Theme, values.String(values.StrLots), "0", false),
+		createOrderBtn:              th.Button(values.String(values.StrBuy)), // TODO: toggle
+		buyOrder:                    true,
+		immediateOrder:              th.CheckBox(new(widget.Bool), values.String(values.StrImmediate)),
+		orderTypesDropdown:          th.DropDown(orderTypes, values.DEXOrderTypes, true),
+		immediateMoreInfo:           th.NewClickable(false),
+		addServerBtn:                th.NewClickable(false),
+		seeFullOrderBookBtn:         th.Button(values.String(values.StrSeeMore)),
+		toggleBuyAndSellBtn:         th.SegmentedControl([]string{values.String(values.StrBuy), values.String(values.StrSell)}),
+		openOrdersBtn:               th.Button(values.String(values.StrOpenOrders)),
+		orderHistoryBtn:             th.Button(values.String(values.StrTradeHistory)),
+		openOrdersDisplayed:         true,
+		ordersTableHorizontalScroll: &widget.List{List: layout.List{Axis: horizontal, Alignment: layout.Middle}},
+	}
+
+	btnPadding := layout.Inset{Top: dp8, Right: dp20, Left: dp20, Bottom: dp8}
+	pg.toggleBuyAndSellBtn.Padding = btnPadding
+	pg.openOrdersBtn.Inset, pg.orderHistoryBtn.Inset = btnPadding, btnPadding
+	pg.openOrdersBtn.Font.Weight, pg.orderHistoryBtn.Font.Weight = font.SemiBold, font.SemiBold
+
+	pg.orderTypesDropdown.SelectedItemDirectionAfterCollapse = layout.E
+	pg.orderTypesDropdown.Width = values.MarginPadding120
+	pg.orderTypesDropdown.FontWeight = font.SemiBold
+	pg.orderTypesDropdown.MakeSelectedItemHoverableAfterCollapse = false
```

Comment:
Is it the selected item you don't want to be hoverable or the collapsed dropdown itself? If it's the dropdown, then you should probably be using a `dropdown.Hoverable` property. Of course, that would only apply to the dropdown, not the dropdown items. The items would continue to be hoverable, unless a use case comes up where that isn't desired and we can introduce a `dropdown.ItemsHoverable` property.